urdf/ur.urdf.xacro

```diff
@@ -16,11 +16,16 @@
    <xacro:arg name="safety_pos_margin" default="0.15"/>
    <xacro:arg name="safety_k_position" default="20"/>
    <!-- ros2_control related parameters -->
-   <xacro:arg name="use_fake_hardware" default="false" />
-   <xacro:arg name="fake_sensor_commands" default="false" />
    <xacro:arg name="headless_mode" default="false" />
    <xacro:arg name="use_tool_communication" default="false" />
 
+     <!-- Simulation parameters -->
+   <xacro:arg name="use_fake_hardware" default="false" />
+   <xacro:arg name="fake_sensor_commands" default="false" />
+   <xacro:arg name="sim_gazebo" default="false" />
+   <xacro:arg name="sim_ignition" default="false" />
+   <xacro:arg name="simulation_controllers" default="$(find ur_bringup)/config/ur_controllers.yaml" />
```

Comment:
I'd prefer not adding the dependency to the driver repo here, as well.

Reply:
OK, should I remove this default value then, or should we copy file here. I am OK with both options.

Reply:
As it is not strictly necessary when using the description standalone, I would be fine with defaulting to an empty string.

Reply:
```suggestion
   <xacro:arg name="simulation_controllers" default="" />
```